docs/sample-out/sections/2020-09-17-15-04-23.csv

```diff
@@ -1,5 +1,5 @@
-﻿SourceSystemIdentifier,SourceSystem,SISSectionIdentifier,Title,SectionDescription,Term,LMSSectionStatus,EntityStatus,CreateDate,LastModifiedDate,SourceCreateDate,SourceLastModifiedDate
-123456780,Google Classroom,25590100102Trad220ALG112011,ALG-1,Algebra I,"255901001_2020_2019-2020_Fall,255901001_2020_2019-2020_Spring",Archived,Active,2020-08-20 12:34:56,2020-08-20 12:34:56
-123456789,Google Classroom,25590100101Trad120ENG112011,ENG-1,English/Language Arts I (9th grade),"255901001_2021_2020-2021_Fall,255901001_2021_2020-2021_Spring",Active,Active,2020-08-20 12:34:56,2020-08-20 12:34:56
-2385758954,Google Classroom,25590100102Trad220ALG112011,ALG-1,Algebra I,"255901001_2021_2020-2021_Fall,255901001_2021_2020-2021_Spring",Active,Active,2020-08-20 12:34:56,2020-08-20 12:34:56
-123456791,Google Classroom,,ENG-STAFF-1,English language arts staff meeting,,Unpublished,Active,9/3/2020 1:02,9/4/2020 0:01
+﻿SourceSystemIdentifier,SourceSystem,SISSectionIdentifier,Title,SectionDescription,Term,LMSSectionStatus,CreateDate,LastModifiedDate,SourceCreateDate,SourceLastModifiedDate
+123456780,Google Classroom,25590100102Trad220ALG112011,ALG-1,Algebra I,"255901001_2020_2019-2020_Fall,255901001_2020_2019-2020_Spring",Archived,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+123456789,Google Classroom,25590100101Trad120ENG112011,ENG-1,English/Language Arts I (9th grade),"255901001_2021_2020-2021_Fall,255901001_2021_2020-2021_Spring",Active,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+2385758954,Google Classroom,25590100102Trad220ALG112011,ALG-1,Algebra I,"255901001_2021_2020-2021_Fall,255901001_2021_2020-2021_Spring",Active,2020-08-20 12:34:56,2020-08-20 12:34:56,,
```

Comment:
These terms have 61 characters and 60 is our max, so this is failing to upload. Can we take the opportunity to fix?

```suggestion
123456780,Google Classroom,25590100102Trad220ALG112011,ALG-1,Algebra I,"255901001_2019-2020_Fall,255901001_2019-2020_Spring",Archived,2020-08-20 12:34:56,2020-08-20 12:34:56,,
123456789,Google Classroom,25590100101Trad120ENG112011,ENG-1,English/Language Arts I (9th grade),"255901001_2020-2021_Fall,255901001_2021_2020-2021_Spring",Active,2020-08-20 12:34:56,2020-08-20 12:34:56,,
2385758954,Google Classroom,25590100102Trad220ALG112011,ALG-1,Algebra I,"255901001_2021_2020-2021_Fall,255901001_2020-2021_Spring",Active,2020-08-20 12:34:56,2020-08-20 12:34:56,,
```